src/connectors/archive.js

```diff
@@ -1,89 +1,28 @@
 'use strict';
 
-const version = $('body').hasClass('navia') ? 'new' : 'legacy';
-switch (version) {
-	case 'new':
-		bindNew();
-		break;
+Connector.currentTimeSelector = '.jw-text-elapsed';
 
-	case 'legacy':
-		bindLegacy();
-		break;
+Connector.durationSelector = '.jw-text-duration';
 
-	default:
-		Util.debugLog('Unable to match version. Please report at https://github.com/web-scrobbler/web-scrobbler/issues', 'warn');
-		break;
-}
+Connector.isPlaying = () => !document.querySelector('video').paused;
 
-function bindCommon() {
-	Connector.currentTimeSelector = '.jw-text-elapsed';
+Connector.playerSelector = '#theatre-ia';
 
-	Connector.durationSelector = '.jw-text-duration';
+Connector.trackArtSelector = '#theatre-ia center > img';
 
-	Connector.isPlaying = () => {
-		return !$('video')[0].paused;
-	};
-}
+Connector.getArtistTrack = () => {
+	let track = document.querySelector('.jwrowV2.playing .ttl, .audio-track-list .selected .track-title').textContent;
```

Comment:
It's better to use array of selectors, as they are much safer.

```js
const artistSelectors = [
	'.key-val-big a span',
	'.metadata-definition > dd',
];
const trackSelectors = [
	'.jwrowV2.playing .ttl',
	'.audio-track-list .selected .track-title',
];

Connector.getArtistTrack = () => {
	let track = Util.getTextFromSelectors(trackSelectors);
	const artist = Util.getTextFromSelectors(artistSelectors);
	...
};
```

Reply:
I think we should document these Util functions more if we want people to adopt them.